go/vt/wrangler/materializer_test.go

```diff
@@ -1162,6 +1162,126 @@ func TestCreateLookupVindexSameKeyspace(t *testing.T) {
 		t.Errorf("same keyspace: got:\n%v, want\n%v", got, want)
 	}
 }
+
+func TestCreateLookupVindexCustomHashing(t *testing.T) {
```

Comment:
Not all vindex types use hashing. IMO, we should use the terminology `CustomVindexType` here, and similar in the PR description and title, and in the new docs.